### PR TITLE
added new version of sequential labs witouht feature columns

### PR DESCRIPTION
**Pull request by takumiohym**
Sub task of #194 

In the new version of this sequential API labs, it doesn't use preprocessing layer itself, but removed feature columns.
In order to do so, changed data from tf.data via `make_csv_dataset()` to simple numpy array dataset.

`make_csv_dataset` creates multiple Tensors, but it cannot be passed to Sequential API model.
In the previous version, DenseFeatures concat multiple Tensor to 1 Tensor to pass it to Sequential API model, but it is not recommended method, so this warning was showing up.
Screen Shot 2022-04-15 at 12 44 27

So in the new version, the order of [2a_dataset_api.ipynb](https://github.com/GoogleCloudPlatform/asl-ml-immersion/blob/1cde15b389189fc6ce5ab2414ced336ab022e7dd/notebooks/introduction_to_tensorflow/solutions/2a_dataset_api.ipynb) and this [3_keras_sequential_api.ipynb](https://github.com/GoogleCloudPlatform/asl-ml-immersion/blob/8de323f806705e90f46c7e7225a87d013eaefde0/notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api.ipynb) can be switched.

### CODE REVIEW

**Comment by sanjanalreddy** [comment]
What do you mean by the "the order of [2a_dataset_api.ipynb](https://github.com/GoogleCloudPlatform/asl-ml-immersion/blob/1cde15b389189fc6ce5ab2414ced336ab022e7dd/notebooks/introduction_to_tensorflow/solutions/2a_dataset_api.ipynb) and this [3_keras_sequential_api.ipynb](https://github.com/GoogleCloudPlatform/asl-ml-immersion/blob/8de323f806705e90f46c7e7225a87d013eaefde0/notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api.ipynb) can be switched" ?

**Comment by takumiohym** [comment]
Thanks for your comment!
I mean Sequential API contents can come before the dataset API as it is just using numpy array dataset.

**Comment by sanjanalreddy** [comment]
I'm also not sure why you have removed the feature columns from this lab if we are not using preprocessing layers yet. The naming of the lab `3_keras_sequential_api_vertex_preprocessing_layers.ipynb` is confusing if we are not using pre-processing layers yet. Does this have something to do with using TF2.8?

**Comment by takumiohym** [comment]
I removed feature_column as it is not the recommended way so far.
I agree with the renaming if it is not confusing for you to have different sufix for this. Just wanted to indicate the version of the class itself.

File: notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex_preprocessing_layers.ipynb [hunk 1]
```diff
@@ -0,0 +1,584 @@
+{
```
Comment (BenoitDherin) [inline notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex_preprocessing_layers.ipynb:1]: <p>Please revert to using a tf.data.Dataset. One learning objective of this lab should be to teach how to connect a tf.data.Dataset with a Keras model. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (BenoitDherin) [inline notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex_preprocessing_layers.ipynb:1]: <p>Wasn't the goal of the lab to use pre-processing layers? I am not seeing them here. Am I missing something?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (BenoitDherin) [inline notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex_preprocessing_layers.ipynb:1]: <p>There is no need to have <code>create_dataset</code> functions if the only thing they do is to create some numpy arrays. These array can be passed directly to the .fit method, making the code simpler. However, as mentioned earlier, one of the main objective of this lab is to learn how to use the tf.data.Dataset with Keras models. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (BenoitDherin) [inline notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex_preprocessing_layers.ipynb:1]: <p>* Maybe we can use <code>model.save</code> instead </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

**Review by BenoitDherin — CHANGES_REQUESTED** [review]
Please revert to using `tf.data.Dataset` since one of the main objective of this lab is to learn how to connect a `tf.data.Dataset` with a Keras model.

Please use the Keras pre-processing layers since it's in the name of this lab, or change the lab name.

**Comment by takumiohym** [comment]
Hi @BenoitDherin,
Thanks for the comment. 
Yes I basically agree that we want to cover Sequential API + tf.data in this lab, but it Sequential API doesn't work with make_csv_dataset as make_csv_dataset output multiple tensor which wee cannot pass to Sequential API.

So if we want to tf.data.Dataset + Sequential API, it should be modify to read CSV without using make_csv_dataset, but with low level modules like TextLineDataset.
I thought it could be a little confusing, but if you think it is better, I am happy to rewrite it.

**Comment by takumiohym** [comment]
@BenoitDherin 
In terms of version, [HashedCrossing layer](https://www.tensorflow.org/api_docs/python/tf/keras/layers/experimental/preprocessing/HashedCrossing) for feature crossing requires tf 2.8, but the other preprocessing layer does work with older versions.
And I usually create a separated instance in 2.6 for MLOps so far. 

The reason I didn't add any preprocessing layer is that the original feature column lab didn't apply any feature engineering as well.

**Comment by takumiohym** [comment]
And sure I will rewrite it with low level tf.data APIs :)

File: notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex_preprocessing_layers.ipynb (hunk 1 above)
Comment (kylesteckler) [reply notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex_preprocessing_layers.ipynb:1]: <p>+1 </p>

**Comment by takumiohym** [comment]
So is it ok to update like this?
- Include tf.data low level API
- Don't include preprocessing layer in this lab so far.
- Remove the suffix from the file name and rename it to `3_keras_sequential_api_vertex.ipynb` (and old version to `3_keras_sequential_api_vertex_feature_columns.ipynb`)

Also, additionally I think it is better to:
- update [2a_dataset_api.ipynb](https://github.com/GoogleCloudPlatform/asl-ml-immersion/blob/master/notebooks/introduction_to_tensorflow/solutions/2a_dataset_api.ipynb) as well to include tf.data low level API section.

**Comment by takumiohym** [comment]
@sanjanalreddy I added tf.data again, and it is ready for review.
After this, I will add this TextLineDataset section to [2a_dataset_api.ipynb](https://github.com/GoogleCloudPlatform/asl-ml-immersion/blob/master/notebooks/introduction_to_tensorflow/solutions/2a_dataset_api.ipynb) for continuity.

**Review by sanjanalreddy — APPROVED** [review]
With the addition of the TextLineDataset, it looks good to me!

**Pull request merged**